RoadMap_Movie/Tests/MockMoviesListPresenter.swift

```diff
@@ -0,0 +1,78 @@
+// MockMoviesListPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import RoadMap_Movie
+import SwiftyJSON
+import XCTest
+
+/// Мок презентера экрана с фильмами
+final class MockMoviesListPresenter: MainScreenPresenterProtocol {
+    // MARK: - Private Properties
+
+    private let networkService: MovieNetworkServiceProtocol
+    private var imageService: ImageServiceProtocol
+    private weak var view: MainScreenProtocol?
+    private var routerProtocol: RouterProtocol
+    private var realmService: RealmServiceProtocol
+    private var keyChainService: KeyChainServiceProtocol
+
+    // MARK: - Public properties
+
+    var movies: [RoadMap_Movie.Movie]?
+    var detail: Details?
```

Comment:
```suggestion
    // MARK: - Public properties

    var movies: [RoadMap_Movie.Movie]?
    var detail: Details?
```
публичные проперти выше приватных